### ISSUE TITLE
"Error: magit-status-jump is not a transient command" when installing github layer

### ISSUE THREAD

**Issue by Kazark** [syl20bnr/spacemacs#13201]
#### Description :octocat:
I encountered an error while installing GitHub layer that makes me think that it is out of sync with the Magit version.

I am happy to provide whatever information is needed or even help out if I can (though an Emacs n00b).

#### Reproduction guide :beetle:
- Start Emacs
- Press `SPC f e d`
- Add `github` to your `dotspacemacs-configuration-layers`
- Press `SPC f e R`
- Observe the errors in `*Compile-Log*`

*Observed behaviour:* :eyes: :broken_heart:
From `*Compile-Log*`:

```
In toplevel form:
forge-bitbucket.el:24:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-commands.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-gitea.el:24:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-github.el:25:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-gitlab.el:25:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-gogs.el:24:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-issue.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-list.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-notify.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-post.el:25:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-pullreq.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-repo.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-revnote.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-semi.el:23:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge-topic.el:27:1:Error: magit-status-jump is not a transient command

In toplevel form:
forge.el:42:1:Error: magit-status-jump is not a transient command
```

*Expected behaviour:* :heart: :smile:
`github` layer installed and usable.

#### System Info :computer:
- OS: darwin
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 3d39b98e9)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion emacs-lisp git github helm markdown org
                 (scala :variables scala-backend 'scala-metals scala-enable-eldoc t scala-use-unicode-arrows t scala-auto-start-backend t)
                 slack spell-checking sql syntax-checking version-control)
```
- System configuration features: JPEG RSVG IMAGEMAGICK GLIB NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS LCMS2

**Comment by BonfaceKilz** [comment]
@Kazark I solved that ^ by installing the latest `transient` from MELPA. Could you try installing it and see if it fixes it for you?

**Comment by Kazark** [comment]
How do I find my Magit version?

**Comment by BonfaceKilz** [comment]
Just do a `M-x magit-version`.

**Comment by Kazark** [comment]
![image](https://user-images.githubusercontent.com/3661051/73199385-c2abcb00-4102-11ea-82ee-e715a0c8c501.png)

**Comment by BonfaceKilz** [comment]
Interesting... You are using an older version of Magit(from 12th Jan). From Melpa's latest build the latest version is `20200126.1750`. Could you try re-installing magit? You can do that by: `M-x package-list`, search for the line you installed magit, then hit `d`(this removes the package. You should find this line in the list of your installed packages), then select the version from MELPA which is un-installed, and hit 'i'(which installs the package). Hit `x` to run the removal and installation.

TL;DR Reinstall the latest magit from MELPA and see if this solves your problem.

After reinstall, confirm that you are using the latest version by hitting: `M-x magit-version`

**Comment by BonfaceKilz** [comment]
Here's the output of my `M-x h o magit-status-jump` (this basically shows you where the command is found):
```
magit-status-jump is an interactive compiled Lisp function in ‘magit-status.el’.

(magit-status-jump)

  This function is for interactive use only.

In a Magit-Status buffer, jump to a section.
```

It appears from the error log you posted, this function is not present in your version of Magit :cry:

**Comment by Kazark** [comment]
@BonfaceKilz that seems to have fixed it. I'll close it for now unless I see later that it hasn't. Thank you!
